### DISCUSSION TITLE
Adding Poe.com as an Opton

### DISCUSSION THREAD

**Discussion by Drakemoor** [logancyang/obsidian-copilot#246] [General]
Hi,

Thanks for the great plugin. 🙇🏻‍♂️

Would you consider adding poe.com as an option? They have great pricing and give access to multiple LLMs for one price. 
I don't know if it's even possible, but I hope it is. 😊

Here is a link to their dev documentation and API thingy, of which I understand nothing. 🙃

Thanks again.

**Comment by logancyang** [comment]
Hi @Drakemoor, thanks for the suggestion! I'll see if this can be added with the current setup.

**Reply by sulth1** [reply]
Hey Logan! Any news about adding Poe? That seem to be the only way to use Claude Opus in Copilot for a reasonable price (20$ per month, instead of the very high price per request). Thanks :)

**Reply by Drakemoor** [reply]
Hi, thanks for the good work you are doing with this plugin. I wonder, have you decided about Poe?

Thanks

**Reply by Drakemoor** [reply]
Poe have added an API for developers. Would you please consider adding it to copilot? 

Here is more info about it: https://creator.poe.com/docs/external-applications/external-application-guide#querying-poe-bots-via-an-api-key

thank you.
